qa/L0_python-self-test/test.sh

```diff
@@ -1,48 +1,20 @@
 #!/bin/bash -e
 # used pip packages
-pip_packages="nose numpy opencv-python tensorflow-gpu torchvision mxnet-cu##CUDA_VERSION##"
```

Comment:
Do we stop testing mxnet here?

Reply:
Yes, there are specific test cases for FW iterators